src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/RichText/RichText.jsx

```diff
@@ -5,18 +5,30 @@ import * as PropTypes from 'prop-types';
 import {FieldPropTypes} from '~/EditPanel/FormDefinitions/FormData.proptypes';
 import {FastField} from 'formik';
 
+function loadOption(selectorOptions, name) {
+    return selectorOptions && selectorOptions.find(option => option.name === name);
+}
+
 export class RichTextCmp extends React.Component {
     constructor(props) {
         super(props);
 
-        CKEditor.editorUrl = window.CKEDITOR_BASEPATH + 'ckeditor.js';
+        const option = props.field.selectorOptions && loadOption(props.field.selectorOptions, 'ckeditor.customConfig');
+
+        if (option) {
+            CKEditor.editorUrl = option.value;
+        } else {
+            CKEditor.editorUrl = window.CKEDITOR_BASEPATH + 'ckeditor.js';
+        }
```

Comment:
The custom configuration must be used by CK Editor, but not replace it.
Here how we set the configuration in GWT:
```
if ((typeof $wnd.CKEDITOR.customConfig) != 'undefined') {
            $wnd.CKEDITOR.tools.extend(cfg,  $wnd.CKEDITOR.customConfig, true);
        } 
```
we should do something similar

Reply:
Thank you